wiki/Tournaments/CWC/1/en.md

```diff
@@ -157,91 +157,322 @@ This competition has come to an end and resulted in the following podium:
+| C | **Team Nub** | **2** | 0 | German Hentai's | [#1](https://osu.ppy.sh/community/matches/2034819) |
+| A | **Indonesia** | **2** | 0 | Lone wolfs | *win by default* |
+
+### Group Stage week 2
+
+Saturday, 12 November 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| --: | --: | :-: | :-: | :-- | :-- |
+| C | **half manual player team** | **2** | 1 | Red Fury | [#1](https://osu.ppy.sh/community/matches/1977802) |
+| D | **China** | **2** | 0 | CatchTheHungarians | [#1](https://osu.ppy.sh/community/matches/1979479) |
+| C | **German Hentai's** | **2** | 0 | Indonesia II | [#1](https://osu.ppy.sh/community/matches/1978912) |
+| A | **Champion The Beat** | **2** | 0 | Indonesia | [#1](https://osu.ppy.sh/community/matches/1978350) |
+
+Sunday, 13 November 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| --: | --: | :-: | :-: | :-- | :-- |
+| B | **CFC** | **2** | 0 | #filipino | [#1](https://osu.ppy.sh/community/matches/1978472) |
+| A | **Viciados em Crack** | **2** | 0 | Lone wolfs | *win by default* |
+
+Monday, 14 November 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| --: | --: | :-: | :-: | :-- | :-- |
+| B | **State(s) of the Art** | **2** | 0 | Old School | [#1](https://osu.ppy.sh/community/matches/1990265) |
+
+### Group Stage week 1
+
+Saturday, 5 November 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| --: | --: | :-: | :-: | :-- | :-- |
+| C | **German Hentai's** | **2** | 0 | half manual player team | [#1](https://osu.ppy.sh/community/matches/1932591) |
+| A | **The Hidden Ones** | **2** | 0 | Viciados em Crack | [#1](https://osu.ppy.sh/community/matches/1934186) |
+
+Sunday, 6 November 2011:
+
+| Group | Team 1 |  |  | Team 2 | Match link |
+| --: | --: | :-: | :-: | :-- | :-- |
+| B | **CFC** | **2** | 0 | Hungary CTB | [#1](https://osu.ppy.sh/community/matches/1930895) |
+| D | **French fruit overdose club** | **2** | 0 | CatchTheHungarians | [#1](https://osu.ppy.sh/community/matches/1942894) |
+| C | **Team Nub** | **2** | 1 | Red Fury | [#1](https://osu.ppy.sh/community/matches/1950105) |
+
+## Ruleset
 
-1. Map order is already given.
-2. Participants will play at 5 maps (maximum) in Quarterfinals, 7 in Semifinals, and 9 in Finals.
+### Tournament rules
+
+1. The Catch the Beat World Cup is a 3v3, team-based tournament played on the osu!catch game mode.
+2. Beatmap scoring is based on **[ScoreV1](/wiki/Score#scorev1)**.
+3. The beatmaps for each round will be announced by the tournament host on the discussion thread in advance before the actual matches take place. Only these beatmaps will be used during the respective matches. 
+   - One beatmap will be a tiebreaker beatmap. This beatmap will only be played in case of a tie.
+4. The match schedule will be settled by the Tournament Management (see the [scheduling instructions](#scheduling-instructions)).
+5. If no staff is available, the match will be postponed.
+6. If a beatmap ends in a draw, the map will be nullified.
```

Comment:
ASC doesn't like the word "map" for some reason; this goes for all other uses of "map" instead of beatmap as well
```suggestion
6. If a beatmap ends in a draw, it will be nullified.
```